tests/mr/device/arena_mr_tests.cpp

```diff
@@ -533,6 +533,42 @@ TEST_F(ArenaTest, Defragment)  // NOLINT
   }());
 }
 
+TEST_F(ArenaTest, PerThreadAndStreamArenas)  // NOLINT
+{
+  auto const arena_size = superblock::minimum_size * 4;
+  arena_mr mr(rmm::mr::get_current_device_resource(), arena_size);
+  std::vector<std::thread> threads;
+  std::size_t num_threads{3};
+  auto view        = std::make_shared<rmm::cuda_stream_view>(rmm::cuda_stream_per_thread);
+  void* thread_ptr = mr.allocate(256, view->value());
```

Comment:
So this one allocates from a thread arena (because it's the PTDS).

Reply:
yes because its cuda_stream_per_thread